### ISSUE TITLE
Update AEST org code

### ISSUE THREAD

**Issue by lmullane** [bcgov/foi-flow#3193]
Title of ticket: 

#### Description
Summarize issue 

AEST needs a new org code as it is now Post-Secondary Education and Skills Training.

#### Dependencies
Are there any dependencies?

Yes, need new org code setup in AXIS.

This task needs to be completed before ministry onboards on January 16th.

#### DOD
- [x] Update org code in FOI Mod
- [x] Map new org code to AEST for reporting

**Comment by sumathi-thirumani-aot** [comment]
Identified Technical Changes
1. Angular App changes
2. Migration scripts
3. Create new internal ministry code (This will avoid bigger technical changes in S3)
**Estimate - 13**

**Comment by lmullane** [comment]
Need another ticket to make is easier to handle org code changes. @sumathi-thirumani-aot to confirm.

**Comment by abin-aot** [comment]
@m-prodan , as discussed I am moving ahead with code "PSE" as of now and as per our earlier discussions - there will be 2 codes displaying in FOI FLOW and but on Angular App( FOI Request Submission) , it will be the new code only!

**Comment by abin-aot** [comment]
TEST users for PSE @lmullane , @m-prodan @KyEggleston @liseandtea . Password is same as with other test users
- foipse@idir

- foipse1@idir

**Comment by KyEggleston** [comment]
@abin-aot 

Confirmed update has been made in Angular App (FOI request submission) in Test. 

When creating a new request in Test-Marshal, I was unable to find the PSE acronym in the 'Select Ministry Client' section:

![image](https://user-images.githubusercontent.com/117312323/210831185-d85b6690-39db-4776-9fdb-62451a4aae80.png)

I was also unable to log in to foipse@idir and foipse1@idir in Test-Marshal, as I got the standard "unauthorized access" message:

![image](https://user-images.githubusercontent.com/117312323/210832146-9c0bd3fb-c059-48d6-a4e3-accf9cdafbbd.png)

**Comment by abin-aot** [comment]
Thanks @KyEggleston , I will check

**Comment by KyEggleston** [comment]
Closing, as 'Ministry Assigned to' queue name has now been updated.

**Comment by abin-aot** [comment]
New group of PSE Ministry Team
![image.png](https://images.zenhubusercontent.com/607de5aafedfab08839427c4/bc8044b5-b8b0-44ed-9806-bf75af534c30)

**Comment by abin-aot** [comment]
PROD RELEASE STEPS Taken
1. Created DB back up of Camunda, FOI DB
	-  patroni-master-002-ffa-camunda_2023-01-10_15-41-16.sql.gz  
	- patroni-master-002-foidb_2023-01-10_15-41-15.sql.gz 
2. Backed images FormsFlowWeb and Requestmanagement api -002
	- oc tag forms-flow-web:prod forms-flow-web:prod-bu
        - oc tag request-management-api:prod-002 request-management-api:prod-002-bu

2. Tagging FormsFlowWeb and Request Managment API - 002
	- oc tag  request-management-api:test request-management-api:prod-002
	- oc tag  forms-flow-web:test forms-flow-web:prod
